SOLUTION4.PY

```diff
@@ -0,0 +1,14 @@
+import csv
+with open("salary.csv",'w',newline='')as f:
+     data=csv.writer(f)
+     header=['sid','sname','salary']
+     record=[[1,'om',4000],
+            [2,'sai',5700],
+            [3,'ram',4200],
+            [4,'ronak',7700],
+            [5,'rekha',7800]]
+     data.writerow(header)
+     data.writerows(record)

```

Comment:
difference between writerow and writerows()?

Reply:
The difference between writerow and writerows is that writerow() will only write one row whereas writerows() will write several rows at once.